staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -318,6 +318,11 @@ func (t *tracker) Add(obj runtime.Object) error {
 	if err != nil {
 		return err
 	}
+
+	if partial, ok := obj.(*metav1.PartialObjectMetadata); ok && len(partial.TypeMeta.APIVersion) > 0 {
+		gvks = []schema.GroupVersionKind{partial.TypeMeta.GroupVersionKind()}
```

Comment:
I'm confused... does a PartialObjectMetadata have APIVersion/Kind set to `metav1.k8s.io/v1`, `PartialObjectMetadata` or to something like `apps/v1`, `Deployment`?

Reply:
sorry, I'm mixing up PartialObjectMetadata and Table output